GitHubConfiguration.ps1

```diff
@@ -910,15 +926,15 @@ function Set-GitHubAuthentication
     }
 
     $script:accessTokenCredential = $Credential
+
+    Write-InvocationLog
```

Comment:
This should be right after the `$PSCmdlet.ShouldProcess()` check, like elsewhere.  Otherwise, we could potentially have the error from line 924 without the invocation line in the log.